frontend/packages/kubevirt-plugin/src/plugin.tsx

```diff
@@ -62,6 +62,18 @@ const plugin: Plugin<ConsumedExtensions> = [
       flag: FLAG_KUBEVIRT,
     },
   },
+  {
+    type: 'NavItem/ResourceNS',
+    properties: {
+      section: 'Workloads',
+      componentProps: {
+        name: 'Virtual Machines Instance',
```

Comment:
isn't the plural of the model better? `Virtual Machine Instances`?

Reply:
Yes :heart:  thanks fixed.

Reply:
I would rather opt for `Virtual Machine Instances` than `Virtual Machines Instances` and thus you could use `VirtualMachineInstanceModel.label`.

I am not a native speaker so please correct me but according to 

- https://english.stackexchange.com/questions/203985/hidden-layer-sizes-vs-hidden-layers-sizes 
- https://english.stackexchange.com/questions/258111/two-plural-nouns-in-a-row?noredirect=1&lq=1

we should drop the `s` if the noun is behaving like an adjective which I think it is.